### PR TITLE
Add processes for operating the PSF CNA

### PR DESCRIPTION
**Pull request by sethmlarson**

### CODE REVIEW

File: docs/python.org/CVE-Numbering-Authority.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
+---
+title: CVE Numbering Authority
+---
+# CVE Numbering Authority
+
+## Points of Contact (POC)
+
+- Seth Michael Larson (`seth@python.org`)
```
Comment (ewdurbin) [inline docs/python.org/CVE-Numbering-Authority.md:8]: I think just names are fine for this doc. Might summarize this by including "All Points of Contact are subscribed to [the CNA email list](mailto:cna@python.org)"

File: docs/python.org/CVE-Numbering-Authority.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+* After filling in all fields, check everything looks accurate in the "Preview" tab.
+* After filling in all fields, send the CVE Record JSON to another CNA operator for review:
+  * In the "Source" tab of Vulnogram copy the JSON document.
+  * Paste the JSON document into the `#cna` PSF Slack channel and ask for a review.
+  * Reviewers can paste the CVE Record JSON into their own Vulnogram instance to review the fields.
+* Once approved, the CVE Record is ready to be published. Select `Post to CVE.org` at the bottom of the Editor tab to immediately publish the CVE Record.
+
+### CVE ID Assignment Requests
+
+We'll receive requests from the `cna at python.org` mailing list requesting CVE IDs.
```
Comment (ewdurbin) [inline docs/python.org/CVE-Numbering-Authority.md:102]: Do we have any requirements as far as responding to the requestor like with disputes/updates?

File: docs/python.org/CVE-Numbering-Authority.md (hunk 1 above)
Comment (ewdurbin) [reply docs/python.org/CVE-Numbering-Authority.md:8]: This applies in later sections that have email addresses as well.

Comment (sethmlarson) [reply docs/python.org/CVE-Numbering-Authority.md]: Done!

File: docs/python.org/CVE-Numbering-Authority.md (hunk 2 above)
Comment (sethmlarson) [reply docs/python.org/CVE-Numbering-Authority.md:104]: I've asked in the CNA Slack channel, it's not clear from the rules.

Comment (sethmlarson) [reply docs/python.org/CVE-Numbering-Authority.md:104]: Got one reply back in the CNA channel which seems to indicate there's no timeline around CVE ID issuance (and that the timeliness rules for CVE Disputes isn't even documented in the rules). We should be good to go with this for now, then once the rules are updated to make everything clear we can update our processes.

**Pull request merged**